lib/samples/apply_function_to_raster_from_file/README.md

```diff
@@ -0,0 +1,45 @@
+# Apply function to raster from file
+
+Apply a raster function to a local raster file and display the output with a raster layer.
+
+![Image of apply function to raster from file](apply_function_to_raster_from_file.png)
+
+## Use case
+
+Raster functions allow processing operations that can be applied to one or more rasters on the fly. A land survey agency may apply hillshade and aspect functions to rasters with elevation data in order to better determine the topography of a landscape and to make further planning decisions.
+
+## How to use the sample
+
+Load the sample to see a raster function applied to a raster.
+
+## How it works
+
+1. Create a `RasterFunction` using the path to a local raster function JSON file.
+2. Set the raster function arguments as required by the function used.
+3. Use the raster function to create a new `Raster`.
+4. Set the raster to a `RasterLayer` and display it in the `Map`.
+
+## Relevant API
+
+* Raster
+* RasterFunction
+* RasterLayer
+
+## Offline data
+
+| Link                                                                                                     | Local Location                                                |
+|----------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|
+| Shasta.tif raster           | `<userhome>`/ArcGIS/Runtime/Data/raster/Shasta_Elevation.tif  |
+| color.json raster function  | `<userhome>`/ArcGIS/Runtime/Data/raster/color.json            |
```

Comment:
This table is written with a desktop application in mind. Those paths don't make sense for a mobile application. In other sample designs where we're given this sort of table, we've replaced it. For example, here: https://github.com/Esri/arcgis-maps-sdk-flutter-samples/blob/v.next/lib/samples/apply_colormap_renderer_to_raster/README.md#offline-data

We should do the same thing here, but with the appropriate portal links.